pkg/signing/verify.go

```diff
@@ -104,3 +104,25 @@ func VerifySegmentID(ctx context.Context, satellite Signee, signed *pb.SatSegmen
 
 	return satellite.HashAndVerifySignature(ctx, bytes, signed.SatelliteSignature)
 }
+
+// VerifyExitCompleted verifies that the signature inside ExitCompleted belongs to the satellite
+func VerifyExitCompleted(ctx context.Context, satellite Signee, signed *pb.ExitCompleted) (err error) {
+	defer mon.Task()(&ctx)(&err)
+	bytes, err := EncodeExitCompleted(ctx, signed)
+	if err != nil {
+		return Error.Wrap(err)
+	}
+
+	return satellite.HashAndVerifySignature(ctx, bytes, signed.ExitCompleteSignature)
```

Comment:
when we return this way, the error won't be wrapped like the above error. is that okay or should we wrap it? same for in VerifyExitFailed

Reply:
Added wrap